sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ClaimsBasedSecurityChannel.java

```diff
@@ -46,6 +48,7 @@ public ClaimsBasedSecurityChannel(Mono<RequestResponseChannel> responseChannelMo
                     final Message request = Proton.message();
                     final Map<String, Object> properties = new HashMap<>();
                     properties.put(PUT_TOKEN_OPERATION, PUT_TOKEN_OPERATION_VALUE);
+                    properties.put(PUT_TOKEN_EXPIRY, Date.from(accessToken.getExpiresAt().toInstant()));
```

Comment:
Would be great to have tests that validate all the properties that are in CBS channel.

Reply:
I'll add one.